releasenotes/notes/solver-generalization-ad588a54c830c66f.yaml

```
@@ -0,0 +1,27 @@
+---
+upgrade:
+  - |
+    The :class:`~qiskit_dynamics.Solver` class has been modified to no longer
```

You can make these sphinx links work without the full path doing `.Solver` or `~.Solver` (assuming the class/function names are unique in the documentation scope)

```suggestion
    The :class:`~.Solver` API has been updated so that ``signals`` are passed
    as a kwarg to the :meth:`.Solver.solve` method rather than being set as a
    class attribute. A new instantiation argument, ``rwa_carrier_freqs``, has been
    added to control behaviour of the rotating wave approximation in the absence
    of signals being specified.
```

Probably dont need to mention the deprecation twice